ricecooker/commands.py

```diff
@@ -292,10 +293,12 @@ def create_tree(tree):
     """
     # Create tree
     config.LOGGER.info("\nCreating tree on Kolibri Studio...")
+    # channel_id, channel_link = tree.upload_channel_structure()
```

Comment:
Im currently committing changes in working segments, and will likely finish the remainder of this part today. I commented it out because that's the code that will need to be run instead of tree.upload_tree(), but the code on the CC side isn't done yet.